test/onnx/test_pytorch_onnx_onnxruntime_cuda.py

```diff
@@ -85,6 +85,20 @@ def forward(self, x):
         model = amp.initialize(LinearModel(), opt_level="O2")
         self.run_test(model, input)
 
+    # ONNX supported bfloat16 for opsets >= 13
+    @skipIfUnsupportedMinOpsetVersion(13)
+    @skipIfNoBFloat16Cuda
+    @disableScriptTest()
+    def test_linear_bfp16(self):
```

Comment:
I'm confused why this is called `linear`?

Reply:
That's my `A100` ITP local linear test name. Fixed it.